packages/@webex/plugin-meetings/src/breakouts/index.ts

```diff
@@ -220,6 +232,59 @@ const Breakouts = WebexPlugin.extend({
 
     this.breakouts.set(Object.values(breakouts));
   },
+
+  /**
+   * Make enable breakout resource
+   * @returns {Promise}
+   */
+  touchBreakout() {
+    // @ts-ignore
+    return this.webex
```

Comment:
shouldn't we check if `this.breakoutServiceUrl` is set before making the request?

Reply:
Ok, i will add it.